templates/resources/grant_privileges_to_account_role.md.tmpl

```diff
@@ -29,6 +29,9 @@ description: |-
 
 {{ .SchemaMarkdown | trimspace }}
 
+## Known limitations
+- Setting the `CREATE SNOWFLAKE.ML.ANOMALY_DETECTION` or `CREATE SNOWFLAKE.ML.FORECAST` privileges on schema results in a permadiff because of the probably incorrect Snowflake's behavior of `SHOW GRANTS ON <object_type> <object_name>`. More in the [comment](https://github.com/Snowflake-Labs/terraform-provider-snowflake/issues/2651#issuecomment-2022634952).
```

Comment:
This should be also included in the docs for snowflake_grant_privileges_to_database_role

Reply:
Hmmm, then, I have to confirm it with the test first. I will check it out and add if this is also true there.

Reply:
Added.